pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -196,6 +196,8 @@ Setting | Default values - Version 2.x | Default values - Version 3.0 | Descript
 `paths.dist.js` | `"./assets/uswds/js"` | `"./assets/uswds/js"` | Project destination for compiled JavaScript
 `paths.dist.css` | `"./assets/uswds/css"` | `"./assets/uswds/css"` | Project destination for compiled CSS
 `sprite.projectIconsOnly` | `false` | `false` | Include _only_ the icons in `paths.src.projectIcons` in the icon sprite
+`settings.compile.browserslist` | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | Define the [browserslist query](https://github.com/browserslist/browserslist?tab=readme-ov-file#queries) for CSS prefixes generated by [autoprefixer](https://github.com/postcss/autoprefixer).
```

Comment:
> [!note] 
> This setting was included in the README but not on the site. I added it here to fill the gap, even though it isn't related to 1.2.0

Reply:
**Praise:** Great catch! Thanks for adding this